src/Administration/Resources/app/administration/src/module/sw-order/component/sw-order-send-document-modal/index.js

```diff
@@ -202,6 +211,14 @@ export default {
                             {
                                 order: this.order,
                                 salesChannel: this.order.salesChannel,
+                                document: this.document,
+                                a11yDocuments: this.a11yDocuments,
+                            },
+                            null,
+                            null,
+                            {
+                                ...Shopware.Context.api,
+                                ...(this.order.languageId && { languageId: this.order.languageId }),
                             },
```

Comment:
This function call looks crazy to me 🙈 . One optional suggestion: Extract that last object into a variable with a proper name. Ideally also all other objects that are passed here